pkg/utils/statsutils.go

```diff
@@ -9,6 +9,20 @@ import (
 	"go.opencensus.io/tag"
 )
 
+func MakeGauge(name, description string, tagKeys ...tag.Key) *stats.Int64Measure {
+	gauge := stats.Int64(name, description, "1")
```

Comment:
nit, i find this a bit easier to read
```suggestion
	gauge := stats.Int64(name, description, stats.UnitDimensionless)
```